text/0000-else.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Else and Else-If Directives
+status: DRAFTED
+created_at: 2022
+---
+
+# Else and Else-If Directives
+
+## Summary
+
+This proposal replaces and enhances the existing `if:true` and `if:else` directives to add directives that express "else" and "else if" logic.

```

Comment:
> This proposal replaces ...

I'm surprised this is a replacement rather than deprecate and offer a better alternative. Offline I'd like to understand the rollout plan for Lightning Platform. cc @pmdartus

Reply:
I don't think this proposal replaces the existing `if:true` and `if:false` directives. As [stated](https://github.com/salesforce/lwc-rfcs/pull/62/files#diff-2556c5c6cd88d674348a2e918cb09e69e6a8f133b8930926d945660d81f31140R262) in the _How we teach this_ section, we would mark the existing directives are legacy. I don't see us dropping support for `if:*` directives in the foreseeable future.

This sentence can certainly be rephrased to alleviate this confusion.

```suggestion
This proposal superseed the existing `if:true` and `if:else` directives to add directives that express "else" and "else if" logic.

```

----

Also on the topic of Salesforce platform enablement, I am wondering if this feature should or not be enabled by default. Should this be gated behind an API version to incentivize developers to bump their component API version's to beneficiate from this feature.

Reply:
Correct, it doesn't "replace" it, it supersedes it.